tasks/setup.py

```diff
@@ -19,7 +19,7 @@
 
 name = 'google-cloud-tasks'
 description = 'Cloud Tasks API API client library'
-version = '0.2.0'
+version = '0.1.0'
```

Comment:
`0.2.0` is [already released](https://pypi.org/project/google-cloud-tasks/0.2.0/).